src/nwbinspector/_nwbinspector_cli.py

```diff
@@ -0,0 +1,212 @@
+)
+@click.option("--overwrite", help="Overwrite an existing report file at the location.", is_flag=True)
+@click.option("--ignore", help="Comma-separated names of checks to skip.")
+@click.option("--select", help="Comma-separated names of checks to run.")
+@click.option(
+    "--threshold",
+    default="BEST_PRACTICE_SUGGESTION",
+    type=click.Choice(["CRITICAL", "BEST_PRACTICE_VIOLATION", "BEST_PRACTICE_SUGGESTION"]),
+    help="Ignores tests with an assigned importance below this threshold.",
+)
+@click.option("--json-file-path", help="Write json output to this location.")
+@click.option("--n-jobs", help="Number of jobs to use in parallel.", default=1)
+@click.option("--skip-validate", help="Skip the PyNWB validation step.", is_flag=True)
+@click.option(
+    "--detailed",
+    help=(
+        "If file_path is the last of 'levels' (the default), identical checks will be aggregated in the display. "
+        "Use '--detailed' to see the complete report."
+    ),
+    is_flag=True,
+)
+@click.option("--progress-bar", help="Set this flag to False to disable display of the progress bar.")
+@click.option(
+    "--modules",
+    help="Modules to import prior to reading the file(s). Necessary for registration of custom checks functions.",
+    type=str,
+    required=False,
+    default=None,
+)
+@click.version_option(__version__)
+def _nwbinspector_cli(
+    *,
+    path: str,
+    stream: bool = False,
+    version_id: Union[str, None] = None,
+    report_file_path: str = None,
+    config: Union[str, None] = None,
+    levels: str = None,
+    reverse: Union[str, None] = None,
+    overwrite: bool = False,
+    ignore: Union[str, None] = None,
+    select: Union[str, None] = None,
+    threshold: str = "BEST_PRACTICE_SUGGESTION",
+    json_file_path: Union[str, None] = None,
+    n_jobs: int = 1,
+    skip_validate: bool = False,
+    detailed: bool = False,
+    progress_bar: Union[str, None] = None,
+    modules: Union[str, None] = None,
+) -> None:
+    """
+    Run the NWB Inspector via the command line.
+
+    Example Usage
+    -------------
+    nwbinspector path/to/folder
+
+    nwbinspector path/to/file.nwb
+
+    nwbinspector 000003 --stream
+
+    nwbinspector 000003:sub-YutaMouse39/sub-YutaMouse39_ses-YutaMouse39-150727_behavior+ecephys.nwb  \
+      --stream --version-id draft
+    """
+    path_is_url = path.startswith("https://")
+    stream = True if path_is_url else stream
+
+    if stream is True and config is not None and config != "dandi":
+        message = (
+            "File content originates from DANDI, but a custom config was specified. "
+            "The 'dandi' config will be used instead."
+        )
+        raise ValueError(message)
+    elif stream is True and config is None:
+        config = "dandi"
+
+    dandiset_version = "draft" if stream is True and version_id is None else version_id
+    handled_config = config if config is None else load_config(filepath_or_keyword=config)
```

Comment:
```suggestion
    handled_config = config or load_config(filepath_or_keyword=config)
```

Reply:
I assume you meant `config = config if config is not None else load_config(filepath_or_keyword=config)`